db/migrations/005_add_ward_id_to_scheduled_calls_table.sql

```diff
@@ -1 +1 @@
-ALTER TABLE scheduled_calls_table ADD ward_id int REFERENCES wards (id);
+ALTER TABLE scheduled_calls_table ADD ward_id INTEGER REFERENCES wards (id);
```

Comment:
I know `int` and `INTEGER` are equivalent, but I'm very wary of making any modifications to migrations that have already been run on production, as this will not be run again.

Can you revert please?